source/draw/Image.ooc

```diff
@@ -27,6 +27,11 @@ CoordinateSystem: enum {
 	YUpward = 0x02
 }
 
+TransformMethod: enum {
```

Comment:
I don't think so.
Ok now that you mentioned it I remember why I don't have a single `resizeTo` with two arguments. It is because all derived classes currently have the single argument version of `resizeTo`. If you remove it and add the two argument version with default second argument, then compiler will complain that all derived image classes should reimplement it. I wanted to keep the changes at minimum.